modules/references/client/components/Interaction.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Interaction(props) {
+
+  const { reference: { interactions } } = props;
+  const isInteraction = [...Object.keys(interactions)].reduce((accumulator, current) => accumulator || interactions[current], false);
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">
+        <h4 id="howDoYouKnowThemQuestion">How do you know them?</h4>
+      </div>
+      <div className="panel-body">
+        <div role="group" aria-labelledby="howDoYouKnowThemQuestion">
+          <div className="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                checked={props.reference.interactions.met}
```

Comment:
`interactions` was extracted from props earlier so we could use the shorter form here too for consistency?